### ISSUE TITLE
Crash on installation

### ISSUE THREAD

**Issue by deNULL** [beefytech/Beef#136]
I've encountered a crash right at the end of installation process (probably when installer is trying to run newly installed IDE?). The IDE is seems to be working okay after that though.

OS: Windows 10.
Crash dump: [beef-install-crash.zip](https://github.com/beefytech/Beef/files/4532929/beef-install-crash.zip)

```
Exception: Access Violation (code 0xc0000005) at address 7ff8'de7500c3 in thread 2710
Executable: C:\Users\deNULL\Downloads\BeefSetup_0_42_2.exe
File Version: 0.42.2
Product Version: 905330ce12cd1f1cb9435463d8ab08c06d6d2451

Logical Address: 0001:'0006f0c3

f2'6776e620 7ff8'de7500c3 BeefInstallUI.dll Start+65913
    Params: '00000050 '000007e4 27c'8dca0000 7ff9'0ed5b997

f2'6776e730 7ff8'de7503f1 BeefInstallUI.dll Start+65C41
    Params: 27c'98b8dea0 7ff9'0ed5fb91 f2'6776e8d0 27c'8dca0000

f2'6776e7a0 7ff8'de74d4df BeefInstallUI.dll Start+62D2F
    Params: 1d61af9'62f102bd 7ff8'de743a80 7ff8'de779b28 7ff8'00000000

f2'6776e7e0 7ff8'de71c1f7 BeefInstallUI.dll Start+31A47
    Params: f2'6776fbf8 f2'6776fb08 f2'6776eab0 f2'6776eaa0

f2'6776fc60 7ff8'de6e5e7f 0001:'00004e7f BeefInstallUI.dll
    Params: '00000000 '00000000 '00000000 '00000002

f2'6776fca0 7ff8'de74b81b BeefInstallUI.dll Start+6106B
    Params: '00000000 '00000000 '00000000 '0000063c

f2'6776fcf0 7ff8'de757f9b BeefInstallUI.dll Start+6D7EB
    Params: 27c'98aaa800 '00000000 '00000000 '00000000

f2'6776fd20 7ff9'0d607bd4 KERNEL32.DLL BaseThreadInitThunk+14
    Params: '00000000 '00000000 '00000000 '00000000

f2'6776fda0 7ff9'0ed8ce51 ntdll.dll RtlUserThreadStart+21
    Params: '00000000 '00000000 '00000000 '00000000


RAX:27c'98b8dea0 RBX:27c'98b8dbd0 RCX:7ff8'de7b8b38 RDX:'000f0027 RSI:'00000000 RDI:'00000000
R8:'0000000f R9:'00000040 R10:'00000000 R11:f2'6776e5a0
R12:19'254d3800 R13:f2'6776ea90 R14:'000007e4 R15:27c'8dd05cd0
EIP:7ff8'de7500c3 ESP:f2'6776e600 EBP:f2'6776e8d0
CS:0033 SS:002B DS:002B ES:002B FS:0053 GS:002B
Flags:'00010206
```

**Comment by bfiete** [comment]
Dup of https://github.com/beefytech/Beef/issues/81. Fix will be in release 0.42.3.

**Comment by HydrogenC** [comment]
Luckily, the crash doesn't affect the installation, the installation will not be currupt.
